person.rb

```diff
@@ -1,9 +1,12 @@
+require 'date'
 require_relative 'nameable'
 require_relative 'capitalize_decorator'
 require_relative 'trimmer_decorator'
+require_relative 'book'
+require_relative 'Rental'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age, :parent_permission
```

Comment:
- [ ] Great job @peterdtitan, everything look fine except that the `Getters` and `Setters` for @rentals should be present in the person class. You have the Getters  method to read the data now, you need to define a  writes data method Setters as well, Please address this 👍🏻 you may add this `attr_writer :rentals` or make both into one line: `attr_accessor :name, :age, :rentals`